torch/_export/serde/serialize.py

```diff
@@ -362,8 +362,12 @@ def deserialize_torch_artifact(serialized: Union[dict[str, Any], tuple[Any, ...]
         return {}
     buffer = io.BytesIO(serialized)
     buffer.seek(0)
-    # weights_only=False as we want to load custom objects here (e.g. ScriptObject)
-    artifact = torch.load(buffer, weights_only=False)
+    # weights_only=False if we want to load custom objects here (e.g. ScriptObject)
+    try :
+        artifact = torch.load(buffer, weights_only=True)
+    except Exception as e:
```

Comment:
except Exception is not the best, you can have a syntax error and it will hit this branch.